mmcv/ops/csrc/common/cuda/gather_points_cuda_kernel.cuh

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) OpenMMLab. All rights reserved
+#ifndef GATHER_POINTS_CUDA_KERNEL_CUH
+#define GATHER_POINTS_CUDA_KERNEL_CUH
+
+#ifdef MMCV_USE_PARROTS
+#include "parrots_cuda_helper.hpp"
+#else
+#include "pytorch_cuda_helper.hpp"
+#endif
+
+#define TOTAL_THREADS 1024
+#define DIVUP(m, n) ((m) / (n) + ((m) % (n) > 0))
```

Comment:
This line appears many times and does it need to be written in a public file?

Reply:
refer to #1337